.circleci/config.yml.in

```diff
@@ -668,7 +668,7 @@ jobs:
     environment:
       <<: *environment
       CUDA_VERSION: 11.7
-      image_name: pytorch/torchaudio_unittest_base:manylinux-cuda10.2-cudnn8-20210623
+      image_name: pytorch/manylinux-cuda117
```

Comment:
This is not a proper fix. Kaldi is missing in the image, and tests will fail.
A proper fix is to build and push test env image with the cuda-11.7 and updating the image to use it.
 https://github.com/pytorch/audio/blob/main/.circleci/unittest/linux/docker/build_and_push.sh

Reply:
Can we use conda to install Kaldi ? https://anaconda.org/conda-forge/kaldi